apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoaderIT.java

```diff
@@ -1247,13 +1226,6 @@ private void validateBeneficiaryAndSkippedCountsInDatabase(
       beneCountQuery.select(criteriaBuilder.count(beneCountQuery.from(Beneficiary.class)));
       Long beneCount = entityManager.createQuery(beneCountQuery).getSingleResult();
       assertEquals(expectedBeneCount, beneCount, "Unexpected number of beneficiary records.");
-
-      // Count and verify the number of bene records in the DB.
-      CriteriaQuery<Long> skippedCountQuery = criteriaBuilder.createQuery(Long.class);
-      skippedCountQuery.select(
-          criteriaBuilder.count(skippedCountQuery.from(SkippedRifRecord.class)));
-      Long skippedCount = entityManager.createQuery(skippedCountQuery).getSingleResult();
-      assertEquals(expectedSkippedCount, skippedCount, "Unexpected number of skipped records.");
```

Comment:
This makes `expectedSkippedCount` obsolete and possibly some tests that call this method as well.

Reply:
yup...rename tests and removed others.